### ISSUE TITLE
Sorting feature

### ISSUE THREAD

**Issue by endurance1968** [edouardparis/lntop#66]
Is there an option of sort especially the channel view per column? E.g. sort per **G**auge level, **L**ocal funds...

**Comment by edouardparis** [comment]
Yes move on the column you want to sort and press `d` (desc) or `a` (asc)

**Comment by nyonson** [comment]
Does this not work for some columns? In the "forwarding" view I don't seem to be able to sort by `Time`.

**Comment by edouardparis** [comment]
the forwarding history does not enable the sort for the time column, It will only display in the recent order.
Do you think it makes sense to implement it ? It can be misleading because only a specific amount of recent forwarding events can be displayed https://github.com/edouardparis/lntop/pull/101
